lib/vmdb/plugins.rb

```diff
@@ -8,6 +8,11 @@ class Plugins
     def initialize
       @registered_automate_domains = []
       @vmdb_plugins = []
+
+      Rails.application.railties.each do |railtie|
+        next unless railtie.class.name.start_with?("ManageIQ::Providers::") || railtie.try(:vmdb_plugin?)
+        register_vmdb_plugin(railtie)
+      end
```

Comment:
Oh, sorry, I mis-read your comment. 

Indeed it would add the engine root *again* to `DescendantLoader.instance.descendants_paths` [here](https://github.com/ManageIQ/manageiq/pull/13941/files#diff-2dffee21730102d591f668045fe85db7L19) - I can fix that by making it a proper method that does not allow double entries. But it makes does no real harm for now...

`register_automate_domain`  only adds new `AutomateDomain` to a newly created array `@registered_automate_domains`